dlrover/python/common/grpc.py

```diff
@@ -238,6 +238,13 @@ class GlobalStep(Message):
     elapsed_time_per_step: float = 0.0
 
 
+@dataclass
+class UserStep(Message):
```

Comment:
An advice:
1. Use ```DiagnosisReportData``` instead of the new definition here.
2. Use 'type' to tag it as 'UserStep' info and do resolving in master.